util/data_set_helpers.py

```diff
@@ -1,8 +1,12 @@
+from __future__ import absolute_import, division, print_function
 
-import threading
-import numpy as np
 import tensorflow as tf
 
+from threading import Thread
+from math import ceil
+from six.moves import range
```

Comment:
Do we really need to import `range` here or can we simply rely on the python 2.x or python 3.x `range`?